src/main/java/pl/Lukasz/Calories/Controller/MealsController.java

```diff
@@ -0,0 +1,66 @@
+package pl.Lukasz.Calories.Controller;
+
+import org.springframework.beans.factory.annotation.*;
+import org.springframework.data.repository.query.Param;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import pl.Lukasz.Calories.Entity.Meals;
+import pl.Lukasz.Calories.Entity.Users;
+import pl.Lukasz.Calories.Service.MealsService;
+import pl.Lukasz.Calories.Service.UsersService;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+
+@RestController
+public class MealsController {
+    @Autowired
+    private MealsService service;
+
+    @GetMapping("/meals")
+    public List<Meals> list(@Param("user") Integer user) {
```

Comment:
Missing error handling for wrong user id, also each error should return a JSON with a human readable message.
e.g. here if wrong id is provided return:
404:
```
{
    "errorMessage": "User with Id 123 does not exist"
}
```
Use spring exception handlers for that